src/dotnet/Chat.UI.Blazor/Components/LeftPanel/LeftPanel.razor

```diff
@@ -5,7 +5,7 @@
 <SideNav
     Side="SideNavSide.Left"
     IsOpen="@_isOpen"
-    VisibilityChanged="@(isOpen => PanelsUI.Left.SetIsVisible(isOpen))">
+    VisibilityChanged="@(NoStateHasChanged.EventCallback<bool>(isOpen => PanelsUI.Left.SetIsVisible(isOpen)))">
```

Comment:
Is this because of some weird side effect of async-like change?

Reply:
It's just to get rid of redundant re-renders.